mms/decoder.go

```diff
@@ -414,7 +414,7 @@ func (dec *MMSDecoder) Decode(pdu MMSReader) (err error) {
 		case X_MMS_RETRIEVE_TEXT:
 			_, err = dec.ReadString(&reflectedPdu, "RetrieveText")
 		case X_MMS_MMS_VERSION:
-			_, err = dec.ReadShortInteger(&reflectedPdu, "Version")
+			_, err = dec.ReadByte(&reflectedPdu, "Version")
```

Comment:
http://www.openmms.org/download/OMA-WAP-MMS.pdf
7.2.18. X-Mms-MMS-Version field says
MMS-version-value = Short-integer

So maybe short integer is doing the wrong thing instead?
http://python-mms.sourceforge.net/api/mms.wsp_pdu.Decoder-class.html#decodeShortInteger